src/PFXImportPowershell/README.md

```diff
@@ -5,18 +5,30 @@ This project consists of helper Powershell Commandlets for importing PFX certifi
 These scripts provide a baseline for the actions that can take place to import your PFX Certificates to Intune. They can be modified and adapted to fit your workflow. Most of the cmdlets are wrappers of Intune Graph calls.
 
 ## What's New?
+
+### Version 1.2
+- Breaking changes:
+	- The global Intune app registration is deprecated and its client ID has been removed from these scripts.  A tenant-specific app registration must be created and its client ID added to your IntunePfxImport.psd1 file.
+	- The previously deprecated Get-IntuneAuthenticationToken command has been removed.  Use Set-IntuneAuthenticationToken instead.  The associated AuthenticationResult parameter has also been removed from the other various commands.
+	- Changed the default redirect uri to https://login.microsoftonline.com/common/oauth2/nativeclient as recommended by Microsoft Azure.  
+- Added the ability to authenticate using a client secret instead of user authentication. This is configured in the app registration and IntunePfxImport.psd1 file.  
+- Switched the underlying authentication library from ADAL (which will soon be unsupported) to MSAL.
+
 ### Version 1.1
 - Added functionality to make private keys exportable, a cmdlet to export the key, and a cmdlet to import a key.
 	- Allows migrating connectors when using the Microsoft Software Key Storage Provider.
 	- Serious security considerations needs to be taken when transferring keys between machines.
 - Deprecated the Get-IntuneAuthenticationToken cmdlet in favore of the new Set-IntuneAuthenticationToken to store the authentication token so that it isn't required as a parameter on every call that interacts with Intune.
 	- Calling Remove-IntuneAuthenticationToken or closing the session is recommended when calls to Intune are complete.
 
-# Building the Commandlets
-## Prerequisite
-Visual Studio 2015 (or above)
+# Configure a Microsoft Azure App Registration
 
-[Graph Permissions](https://developer.microsoft.com/en-us/graph/docs/concepts/permissions_reference) required:
+An app registration must be configured for your tenant.  Create the app registration in the Microsoft Azure portal.
+
+Set the redirect uri for the Public client/native (mobile & desktop) platform to https://login.microsoftonline.com/common/oauth2/nativeclient
```

Comment:
I guess this answers my question above. :)

Reply:
correct